### PR TITLE
JAMES-2549 Set default consistency level to Quorum

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java [hunk 1]
```diff
@@ -186,8 +184,18 @@ public Cluster build() {
...
+                .setConsistencyLevel(ConsistencyLevel.QUORUM);
+
+        getRefreshSchemaIntervalMillis().ifPresent(refreshSchemaIntervalMillis ->
+                    queryOptions.setRefreshSchemaIntervalMillis(refreshSchemaIntervalMillis));
+
+        return queryOptions;
+    }
+
     private Optional<Integer> getRefreshSchemaIntervalMillis() {
-        return forTest ? Optional.of(0) : refreshSchemaIntervalMillis;
```
Comment (mbaechler) [inline backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java]: why did you removed that ?

Comment (aduprat) [reply backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java]: This feature was buggy & is not used (the _ClusterBuilder.refreshSchemaIntervalMillis_ method is never called).
Do you want that I remove it, I hesitate on that point?

I think it was used in order to optimize the tests previously.

Comment (mbaechler) [reply backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java]: I would start the PR with a commit containing only that part and a comment that explains it's not in use

**Comment by aduprat** [comment]
push -f

File: backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java (hunk 1 above)
Comment (rouazana) [reply backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java]: so forTest seems to not be used anymore, can you remove it?

Comment (aduprat) [reply backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java]: That was my question...

**Comment by aduprat** [comment]
```

[ERROR] Errors: 
[ERROR]   RabbitMQClusterTest$ClusterSharing.setup:82 » IO
```
test this please

**Pull request merged**